easybuild/easyconfigs/g/gff3tool/gff3tool-2.1.0-foss-2022a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'gff3tool'
+version = '2.1.0'
+
+homepage = 'https://gff3toolkit.readthedocs.io'
+description = 'Python programs for processing GFF3 files'
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = ['gff3tool-2.1.0.tar.gz']
+patches = ['GFF3toolkit-2.1.0-remove_hardcode.patch']
+checksums = [
+    {'gff3tool-2.1.0.tar.gz': 'e842ef4afdb66926a2b63bf2619c7ef30bd9b76a5f1084a5c57bf8b39cc4f15d'},
+    {'GFF3toolkit-2.1.0-remove_hardcode.patch': '03696635af25ab1e3d650b46c7aeaaaadb31e7eac766039c758150f3f3162cb7'},
+]
+
+local_blast_path = '%(installdir)s/lib/python%(pyshortver)s/site-packages/gff3tool/lib/ncbi-blast+'
+preinstallopts = 'mkdir -p %s && ' % local_blast_path
+preinstallopts += 'ln -s $EBROOTBLASTPLUS/bin  %s && ' % local_blast_path
+
+dependencies = [
```

Comment:
Move this up, below `checksums`, above `local_blast_path`

Reply:
`local_blast_path` is defined for `preinstallopts` so I believe it should stay above them. Having `preinstallopts` above would result in parsing error in Easybuild.